### PR TITLE
include twophase gaswater in the test suite

### PR DESCRIPTION
**Pull request by plgbrts**
I am not sure if `add_test_compare_parallel_simulation` is needed here.

### CODE REVIEW

**Comment by akva2** [comment]
it certainly won't hurt to run the parallel test as well.

**Comment by akva2** [comment]
jenkins build this please

File: compareECLFiles.cmake [hunk 1]
```diff
@@ -1007,6 +1014,13 @@ if(MPI_FOUND)
                                        REL_TOL ${rel_tol_parallel}
                                        TEST_ARGS --linear-solver-reduction=1e-7 --tolerance-cnv=5e-6 --tolerance-mb=1e-8)
 
+  add_test_compare_parallel_simulation(CASENAME spe1_gaswater
+                                       FILENAME SPE1CASE2_GASWATER
+                                       SIMULATOR flow
+                                       ABS_TOL ${abs_tol_parallel}
+                                       REL_TOL ${rel_tol_parallel}
+                                       TEST_ARGS --linear-solver-reduction=1e-7 --tolerance-cnv=5e-6 --tolerance-mb=1e-8)
```
Comment (akva2) [inline compareECLFiles.cmake:1022]: this needs DIR

**Comment by akva2** [comment]
jenkins build this update_data please

File: compareECLFiles.cmake (hunk 1 above)
Comment (plgbrts) [reply compareECLFiles.cmake:1023]: Added DIR. Why do certain cases require the specification of DIR and others not?

Comment (akva2) [reply compareECLFiles.cmake:1023]: the default is that DIR is the same as the test name.

**Comment by akva2** [comment]
jenkins build this opm-tests=575 please

**Pull request merged**